packages/create-jest/src/runCreate.ts

```diff
@@ -28,9 +31,36 @@ const {
 
 const getConfigFilename = (ext: string) => JEST_CONFIG_BASE_NAME + ext;
 
-export default async function init(
-  rootDir: string = tryRealpath(process.cwd()),
+export async function runCLI(): Promise<void> {
+  try {
+    const argv = await yargs(process.argv.slice(2))
+      .usage('Usage: $0 [rootDir]')
+      .version(getVersion())
```

Comment:
If it is decided to keep `yargs`, this should be `create-jest` version. Or?

Reply:
Removed `yargs` to keep package lighter